support/docker/README.md

```diff
@@ -18,10 +23,97 @@ To kill the server:
 
     docker-compose down
 
+If you wish to run a bot in the container after it's started:
+
+    docker exec --user npserver -it netpanzer-server \
+        netpanzer -b localhost:3031 --game_config=.netpanzer/config/bot-khan.cfg
+
 Also see the [docker compose
 docs](https://docs.docker.com/compose/features-uses/) for more options and
 ways to edit the compose yml in this directory.
 
 You may alternatively wish to use from a specific release. You may edit the
-compose file to use a differet tag. The images are at
-https://github.com/netpanzer/netpanzer/pkgs/container/netpanzer
+compose file to use a different tag.
+
+## Playing from a container
```

Comment:
@winrid My thoughts on this section are that it's low priority and may be unneeded, but it might be useful for testing. We may decide based on feedback as we get more users and devs whether or not we keep it. What do you think?

Reply:
Personally I would be surprised to see it used when you have the existing binaries and (hopefully) web version. Just from a glance it looks like a lot of work. Do you use it for testing? Usually when testing I just test on the target platform, because who knows when you run into docker-specific issues or you're not properly testing the target platform.

Reply:
*lot of work for the player

Reply:
I just used it once for testing but I don't anticipate i'd need it too often, if at all. I'll remove it.